components/Participants.js

```diff
@@ -0,0 +1,24 @@
+import { styled } from "styled-components";
+
+export default function SelectedParticipants({ participants, handleDeselect }) {
+  return (
+    <ul>
+      {participants.map((participant) => (
+        <FlexContainer key={participant}>
+          <StyledListItem>{participant}</StyledListItem>
```

Comment:
your direct ul children must be the li elements. You can remove the Flexcontainer and add the css to your StyledListItem